SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -324,4 +326,7 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	  */
 	UPROPERTY(Config)
 	bool bEnableMultiWorkerDebuggingWarnings;
+
+	UPROPERTY(EditAnywhere, Config, Category = "Logging", AdvancedDisplay, meta = (DisplayName = "Whether or not to suppress a warning if an RPC of Type is being called with unresolved references. Default is false."))
```

Comment:
Can we add a line that when this is use, QueuedIncomingWaitRPC time is still respected in the case of unresolved references?